src/platform/ti/cc13xx_26xx/cc13x4_26x4/BLEManagerImpl.h

```diff
@@ -110,6 +114,7 @@ typedef struct
 {
     uint8_t len;  // data length
     void * pData; // pointer to message
+
```

Comment:
Sort of nitpicking but should this just be removed?

Reply:
removed